completion/available/docker-machine.completion.bash

```diff
@@ -1,5 +1,20 @@
 #
 # bash completion file for docker-machine commands
+
```

Comment:
Docker-Machine is deprecated, the recommendation is to move to native Docker apps on Windows/Mac. This might be a candidate for deprecation...

Reply:
I think this can be a separate issue, as this PR is big enough as it is

Reply:
see #1744